crates/sui-bridge-indexer/src/migrations/2024-09-05-180103_governance_actions/up.sql

```diff
@@ -1,6 +1,8 @@
 CREATE TABLE governance_actions
 (
     id                          BIGSERIAL    PRIMARY KEY,
+    nonce                       BIGINT,
+    data_source                 TEXT         NOT NULL,
```

Comment:
we should use chain_id v.s. data_source for convention

Reply:
I was following how we are currently doing it for the "token transfer" table.

Reply:
chain id and datasource is 2 different thing, chain id is part of the key of the bridge message and the datasource in token transfer table is referring to the source of the data.

Reply:
in this case I think chain id is more appropriate, we might also want to include the message type as well.

Reply:
do you plan to want this to chain_id in this PR ? @patrickkuo

Reply:
we will need to include the chain id in the event to be able to record it, which require more changes, will need to leave it post launch